src/backend/vulkan/src/lib.rs

```diff
@@ -646,6 +652,7 @@ pub struct PhysicalDevice {
     handle: vk::PhysicalDevice,
     extensions: Vec<vk::ExtensionProperties>,
     properties: vk::PhysicalDeviceProperties,
+    known_memory_flags: vk::MemoryPropertyFlags,
```

Comment:
Could this be a `const`?

Reply:
nope, ash's bitflags are not const or-able

Reply:
You could convert to raw and back, though that's a bit ugly.